src/odemis/gui/comp/canvas.py

```diff
@@ -983,7 +983,9 @@ def _draw_merged_images(self, ctx, interpolate_data=False):
                     #     1         0      1
                     # TODO: for now, this only works correctly if the background
                     # is black (otherwise, the background is also mixed in)
-                    if bm_last == BLEND_SCREEN:
+                    if 'merge_ratio' in md:
+                        merge_ratio = md['merge_ratio']
```

Comment:
It is worthy a comment, because no code in miccanvas or dblmiccanvas sets `'merge_ratio'`. So it can look a little bit mysterious on when this is used. So please explain this is some sort of hack to override the general `.merge_ratio` per image.